### PR TITLE
TP2000-383 Add new workbasket pages from design

### PR DESCRIPTION
**Pull request by eadpearce**
# TP2000-383 Add new workbasket pages from design

## Why

* Separates out some of the workbasket info into different pages

## What

Screenshot 2022-07-11 at 15 00 41

### CODE REVIEW

File: workbaskets/views/ui.py [hunk 1]
```diff
@@ -56,7 +56,13 @@ def get_queryset(self):
...
 
 
-class WorkBasketCreate(DraftCreateView):
+class WorkBasketConfirmCreate(DetailView):
+    template_name = "workbaskets/confirm_create.jinja"
+    model = WorkBasket
+    queryset = WorkBasket.objects.all()
+
+
+class WorkBasketCreate(CreateView):
```
[the remark this answers is not in the document]
Comment (eadpearce) [reply workbaskets/views/ui.py:65]: would that be "workbaskets.change_workbasket" ?

File: common/urls.py [hunk 2]
```diff
@@ -15,7 +15,21 @@
 
 urlpatterns = [
     path("dashboard", views.DashboardView.as_view(), name="dashboard"),
-    path("my-workbasket/", views.MyWorkbasketView.as_view(), name="my-workbasket"),
+    path(
+        "preview-workbasket/",
```
[the remark this answers is not in the document]
Comment (paulpepper-trade) [reply common/urls.py:19]: Ah, the user's active WorkBasket. Right.

File: workbaskets/views/ui.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulpepper-trade) [reply workbaskets/views/ui.py:65]: Yes, I suspect so.

File: workbaskets/views/ui.py [hunk 3]
```diff
@@ -260,3 +277,175 @@ def download_envelope(request):
     )
 
     return HttpResponseRedirect(url)
+
+
+@method_decorator(require_current_workbasket, name="dispatch")
+class EditWorkbasketView(DashboardView):
+    template_name = "common/edit-workbasket.jinja"
```
Comment (paulpepper-trade) [inline workbaskets/views/ui.py:284]: Now that we've moved the view classes across to this module, we should probably do the same for their templates.

**Review by paulpepper-trade — APPROVED** [review]

**Pull request merged**